src/main/java/com/assertthat/selenium_shutterbug/core/PageSnapshot.java

```diff
@@ -51,7 +51,7 @@ public PageSnapshot highlight(WebElement element) {
      */
     public PageSnapshot highlight(WebElement element, Color color, int lineWidth) {
         try {
-            image = ImageProcessor.highlight(image, new Coordinates(element, devicePixelRatio), color, lineWidth);
+            ImageProcessor.highlight(image, new Coordinates(element, devicePixelRatio), color, lineWidth);
```

Comment:
Need to keep assignment to image so subsequent chained methods can use updated image.